plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestPinotConnectorTest.java

```diff
@@ -0,0 +1,379 @@
+                    .set("nationkey", row.getField(3))
+                    .set("phone", row.getField(4))
+                    .set("acctbal", row.getField(5))
+                    .set("mktsegment", row.getField(6))
+                    .set("comment", row.getField(7))
+                    .set("updated_at", initialUpdatedAt.plusMillis(1000).toEpochMilli())
+                    .build()));
+        }
+        kafka.sendMessages(customerRowsBuilder.build().stream(), schemaRegistryAwareProducer(kafka));
+        pinot.createSchema(getClass().getClassLoader().getResourceAsStream("customer_schema.json"), customerTableName);
+        pinot.addRealTimeTable(getClass().getClassLoader().getResourceAsStream("customer_realtimeSpec.json"), customerTableName);
+    }
+
+    @SuppressWarnings("DuplicateBranchesInSwitch")
+    @Override
+    protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
+    {
+        switch (connectorBehavior) {
+            case SUPPORTS_INSERT:
+            case SUPPORTS_DELETE:
+            case SUPPORTS_UPDATE:
+            case SUPPORTS_MERGE:
+                return false;
+
+            case SUPPORTS_TOPN_PUSHDOWN:
+                return false;
+
+            case SUPPORTS_CREATE_SCHEMA:
+                return false;
+
+            case SUPPORTS_CREATE_TABLE:
+            case SUPPORTS_RENAME_TABLE:
+                return false;
+
+            case SUPPORTS_ADD_COLUMN:
+            case SUPPORTS_RENAME_COLUMN:
+            case SUPPORTS_SET_COLUMN_TYPE:
+                return false;
+
+            case SUPPORTS_COMMENT_ON_TABLE:
+            case SUPPORTS_COMMENT_ON_COLUMN:
+                return false;
+
+            case SUPPORTS_CREATE_VIEW:
+            case SUPPORTS_CREATE_MATERIALIZED_VIEW:
+                return false;
+
+            case SUPPORTS_ARRAY:
+            case SUPPORTS_ROW_TYPE:
+                return false;
+
+            default:
+                return super.hasBehavior(connectorBehavior);
+        }
+    }
+
+    @Override
+    protected MaterializedResult getDescribeOrdersResult()
+    {
+        return resultBuilder(getSession(), VARCHAR, VARCHAR, VARCHAR, VARCHAR)
+                .row("updated_at_seconds", "bigint", "", "")
+                .row("clerk", "varchar", "", "") // String columns are reported only as varchar
+                .row("comment", "varchar", "", "")
+                .row("custkey", "bigint", "", "") // Long columns are reported as bigint
+                .row("orderdate", "date", "", "")
+                .row("orderkey", "bigint", "", "")
+                .row("orderpriority", "varchar", "", "")
+                .row("orderstatus", "varchar", "", "")
+                .row("shippriority", "integer", "", "")
+                .row("totalprice", "double", "", "")
+                .build();
+    }
+
+    @Test
+    @Override
+    public void testShowColumns()
+    {
+        assertThat(query("SHOW COLUMNS FROM orders")).matches(getDescribeOrdersResult());
```

Comment:
Can we leave a code comment why overriding this test? Same for testSelectInformationSchemaColumns.

Reply:
I left the same comment above each method that failed without it - lmk if it's ok to just leave in `getDescribeOrdersResult`.